arch/arm/soc/unisoc_uwp/uwp3/Kconfig.series

```diff
@@ -0,0 +1,11 @@
+# Kconfig.series - UNISOC UWP3
+#
+
+config  SOC_SERIES_UWP3
+	bool "Unisoc UWP3"
+	select CPU_CORTEX_M
```

Comment:
Not needed to specify CPU_CORTEX_M, if you have already selected CPU_CORTEX_M4

Reply:
Thanks for the review, patch has been updated.